### DISCUSSION TITLE
alist挂载天翼云盘，正常使用两年，11月20日开始出现用aria2c下载如不用-o参数会出现下载文名不正常的情况

### DISCUSSION THREAD

**Discussion by gardenocean** [AlistGo/alist#7541] [General]
我使用alist挂载天翼云盘，已正常使用两年。当前版本为3.16.3   。但从11月20日开始，出现用aria2c下载时，如不用-o参数会出现下载文名不正常的情况。例如：aria2c -c http://XXX.XXX.XXX.XXX:5244/d/tianyipc/Mac/6666.dmg?sign=i92s6QTokaOAxATSVDIn1fGUkxS06lsLmh7JODqscW8=:0
，下载后会得到一个名为d7936dda-d8a4-4e72-b599-acb39ae7d9d8.dmg的文件，而不是正确的文件名6666.dmg

**Comment by gardenocean** [comment]
这个问题没有人回答吗？

**Comment by pongfcnkl** [comment]
开web代理

**Comment by gardenocean** [comment]
谢谢。今天在没开web代理的情况下，我又用aria2c试了一下同样连接的下载，没用使用-o参数，下载后的文件名又正常了。估计是天翼云盘后台在20日进行了什么调整导致的。现在天翼云盘后台又调整好了。

**Comment by loonghfut** [comment]
目前我的天翼云似乎挂载不上了
报错：connect: no route to host
